webapp/lib/view.dart

```diff
@@ -500,161 +475,186 @@ class BatchRepliesConfigurationView extends PackageConfiguratorView {
+    var tagElement = new DivElement()
+      ..classes.add('tag')
+      ..dataset['id'] = tag.isEmpty ? 'id-123' : tag;
+
+    switch (tagStyle) {
+      case model.TagStyle.Important:
+        tagElement.classes.add('tag--important');
+        break;
+      default:
+        break;
+    }
+
+    var tagText = new SpanElement()
+      ..classes.add('tag__name')
+      ..text = tag
+      ..title = tag;
+
+    tagElement
+      ..append(tagText)
       ..append(
-        new ButtonElement()
-          ..classes.add('button-remove-conversation-responses')
+        new SpanElement()
+          ..classes.add('tag__remove-btn')
           ..text = 'x'
-          ..onClick.listen((event) => (event.target as Element).parent.remove()) // TODO: call controller.command
-      )
-      ..append(
-        new ParagraphElement()
-          ..classes.add('conversation-response__language')
-          ..text = suggestedResponse != null ? suggestedResponse['messages'][0] : ''
-          ..contentEditable = 'true'
-      )
-      ..append(
-        new ParagraphElement()
-          ..classes.add('conversation-response__language')
-          ..text = suggestedResponse != null ? suggestedResponse['messages'][1] : ''
-          ..contentEditable = 'true'
-      )
-      ..append(
-        DivElement()
-          ..classes.add('conversation-response__reviewed')
-          ..append(
-            new CheckboxInputElement()
-                ..classes.add('conversation-response__reviewed-state')
-                ..checked = suggestedResponse != null ? suggestedResponse['reviewed'] : false
-                ..onClick.listen((event) => _reviewSuggestedReplies(event))
-          )
-          ..append(
-            new ParagraphElement()
-            ..classes.add('conversation-response__reviewed-description')
-            ..text = suggestedResponse != null ? '${suggestedResponse['reviewed-by']}, ${suggestedResponse['reviewed-date']}' : ''
-          )
+          ..onClick.listen((_) {
+            if (isEditableTag) {
+              onTagChangedCallback(tag, tag, tagStyle);
+            } else {
+              onTagChangedCallback(tag, tagStyle);
+            }
+          })
       );
-  }
 
-  void _reviewSuggestedReplies(MouseEvent event) {
-    // TODO: call controller.command()
-    var reviewCheckbox = (event.target as CheckboxInputElement);
-    var reviewDescription = reviewCheckbox.nextElementSibling;
-    if (reviewCheckbox.checked) {
-      var reviewedBy = controller.signedInUser.userEmail;
-      var now = DateTime.now().toLocal();
-      var reviewedDate = '${now.year}-${now.month}-${now.day}';
-      reviewDescription.text = '${reviewedBy}, ${reviewedDate}';
-    } else {
-      reviewDescription.text = '';
+    if (isEditableTag) {
+      tagText.contentEditable = 'true';
+      tagText.focus(); // HACK: this is looking a bit odd - if the user moves the cursor at the end of the text box
```

Comment:
this is no longer happening (ie. the new tag isn't focused on anymore after it's created) - can you please find out why?

Reply:
Fixed it